packages/hash/README.md

```diff
@@ -99,24 +82,15 @@ To run HASH locally, please follow these steps:
 
     See `package.json` → `scripts` for details and more options.
 
----
-
-To stop Docker containers with external services, run:
-
-```sh
-yarn external-services down
-```
-
-Container data is persisted locally inside `var/external-services`.
-You can delete this directory when containers are stopped for a ‘hard reset’.
-
 ## User authentication
 
 Development users are seeded when the HASH API is started, these users are `alice@example.com` and `bob@example.com`.
 You'll be able to sign in to these users with the password `password`.
 
 ## Sending emails
 
+[//]: # (TODO: **Disclaimer: Current flux of the repo means the code mentioned in this section is temporarily disabled**)
```

Comment:
Generally how do we want to handle this? I think we should just have one global disclaimer at the moment stating that the repo is in flux and features may be broken, or unused. (For instance the guidance around block development will need a hefty disclaimer that might be quite difficult to keep in sync with development if we intend it to be comprehensive).

cc @nonparibus @CiaranMn

Reply:
Provided we continue tracking the broken bits and pieces that will need fixing following the merge of `dev/graph` into `main`, I'm happy to lose the section-by-section disclaimers and replace it with a single one up top.

Let's still tell users what they should and shouldn't expect to work in the app, however, by writing a disclaimer along the lines of "We are currently migrating the HASH app to a new backend and the `main` branch is unstable. Certain functionality may be broken or disabled, including: X, Y, Z".

Reply:
Ciaran has suggested similar here: https://github.com/hashintel/hash/pull/1177#discussion_r990060824

Reply:
I've taken a go at this: https://github.com/hashintel/hash/pull/1177/commits/efa45283d11a23392cceb30198c644e115564a8f

Please feel free to push commits / suggest changes at will (please do so through the files tab rather than clicking on that commit link because GitHub is confusing)